docs/User-Guide.md

```diff
@@ -453,7 +456,64 @@ Outputs are additionally logged to /opt/intelmq/var/log/intelmqctl
 
 #### Botnet Concept
 
-The botnet represents all currently configured bots. To get an overview which bots are running, use `intelmqctl status`or use IntelMQ Manager.
+The botnet represents all currently configured bots which are explicitly enabled. To get an overview which bots are running, use `intelmqctl status` or use IntelMQ Manager. Set `"enabled": true` in the runtime configuration to add a bot to botnet. By default, bots will be configured as `"enabled": false` therefore, does not belong to the botnet due protection reasons. See [Bots](Bots.md) for more details on configuration.
+
+Disabled bots can still be started explicitly using `intelmqctl start <bot_id>`, but will keep marked as disabled if stopped. They are not started by `intelmqctl start` in analogy to the behavior of widely used initialization systems.
```

Comment:
"but will keep marked" --> "but will remain in the state `disabled` if stopped (and not be implicitly enabled by the `start` command).